unpack_files.py

```diff
@@ -0,0 +1,47 @@
+"""
+shared-latent-space/unpacked_files.py
+
+This function will unpack the data from their compressed form
+
+
+Author: Chris Williams
+Date: 5/23/18
+"""
+
+import os
+import cPickle
+import gzip
+
+import gzip
+import cPickle
+import shutil
+
+
+def unpackFiles(name):
```

Comment:
What are you doing in this function? Are you reading and writing files simultaneously? if so I highly recommend this change:
```python
train_file_to_read = os.path.join ...
train_file_to_write = os.path.join...
test_file_to_read = os.path.join...
test_file_to_write = os.path.join...

with gzip.open(train_file_to_read) as g, gzip.open(train_file_to_write) as h:
  ... do your operations

with gzip.open(test_file_to_read) as g, gzip.open(test_file_to_write) as h:
   ... do operations like above
```

Reply:
That definitely makes it a lot clearer.